docs/components/featured-content-modules.md

```diff
@@ -52,6 +46,29 @@ variation_groups:
           * Visual is always in a 16:9 ratio
 
           * 15px of left, right, and bottom padding around all content in the FCM
+        variation_implementation: ""
+        variation_code_snippet: >-
+          <section class="o-featured-content-module
+                          o-featured-content-module__center">
+              <div class="o-featured-content-module_text">
+                  <h2>Featured content</h2>
+                  <p>
+                      Lorem ipsum dolor sit amet, ei ius adhuc inani iudico, labitur
+                      instructior ex pri. Cu pri inani constituto, cum aeque noster
+                      commodo cu.
+                  </p>
+                  <a class="a-link
+                            a-link__jump
+                            a-link__icon-after-text">
+                      <span class="a-link_text">Read more about the feature</span>
+                  </a>
+              </div>
+              <div class="o-featured-content-module_visual">
+                  <img class="o-featured-content-module_img"
+                      src="https://files.consumerfinance.gov/f/images/cfpb_adult_financial_education_featured.original.png"
```

Comment:
Should we use https://dummyimage.com instead?

Reply:
Prrrooooobably 👍